core/src/main/java/io/grpc/Status.java

```diff
@@ -182,7 +182,8 @@
     /**
      * The service is currently unavailable.  This is a most likely a
      * transient condition and may be corrected by retrying with
-     * a backoff.
+     * a backoff. Note that it is not always safe to retry
+     * non-idempotent operations.
```

Comment:
Can we reword to avoid double negation? Such as:

"Note that it could be unsafe to retry non-idempotent operations" 

or even better

"Note that it is safe to retry only idempotent operations."